operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -0,0 +1,299 @@
+      ),
+      // Node Info
+      (
+        panel.new('Node Info', 'table') +
+        panel.withDescription(|||
+          Information about a specific cluster node.
+        |||) +
+        panel.withPosition({ x: 0, y: 1, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newNamedInstantQuery(
+            expr='sum(cluster_node_lamport_time{instance="$instance"})',
+            refId='Lamport clock time',
+            format='table',
+          ),
+          panel.newNamedInstantQuery(
+            expr='sum(cluster_node_update_observers{instance="$instance"})',
+            refId='Internal cluster state observers',
+            format='table',
+          ),
+          panel.newNamedInstantQuery(
+            expr='sum(cluster_node_gossip_health_score{instance="$instance"})',
+            refId='Gossip health score',
+            format='table',
+          ),
+          panel.newNamedInstantQuery(
+            expr='sum(cluster_node_gossip_proto_version{instance="$instance"})',
+            refId='Gossip protocol version',
+            format='table',
+          ),
+        ]) +
+        panel.withTransformations([
+          {
+            id: 'renameByRegex',
+            options: {
+              regex: 'Value #(.*)',
+              renamePattern: '$1',
+            },
+          },
+          {
+            id: 'reduce',
+            options: {},
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              indexByName: {},
+              renameByName: {
+                Field: 'Metric',
+                Max: 'Value',
+              },
+            },
+          },
+        ])
+      ),
+      // Gossip ops/sec
+      (
+        panel.new('Gossip ops/s', 'timeseries') +
+        panel.withDescription(|||
+          Gossip ops/sec for this node.
+        |||) +
+        panel.withPosition({ x: 12, y: 1, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_node_gossip_received_events_total{instance="$instance"}[$__rate_interval])',
+            legendFormat='{{event}}'
+          ),
+        ]) +
+
+        panel.withOptions({
+          legend: {
+            calcs: [],
+            displayMode: 'list',
+            placement: 'bottom',
+            showLegend: true,
+          },
+          tooltip: {
+            mode: 'single',
+            sort: 'none',
+          },
+        })
+      ),
+      // Known peers
+      (
+        panel.new('Known peers', 'stat') +
+        panel.withDescription(|||
+          Known peers to the node (including the local node).
+        |||) +
+        panel.withPosition({ x: 0, y: 9, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='sum(cluster_node_peers{instance="$instance"})',
+          ),
+        ]) +
+        panel.withFieldConfigs({
+          defaults: {
+            unit: 'suffix:peers',
+          },
+        })
```

Comment:
Let's keep this as-is, or extract it in a `withUnit` method? Similarly, we could use a `withOverrides` below instead of passing the entire struct.

Reply:
Oh, you'd already added `withUnit`. I added `withOverrides` in a similar fashion and removed the generic `withFieldConfig` altogether.